mkdocs.yml

```diff
@@ -33,19 +33,34 @@ theme:
 nav: # Navigation structure for the documentation
   - Home: index.md
   - What's new: whatsnew.md
-  - Get started: get-started.md
-  - Tutorials: # this creates a section in the TOC
-    - notebooks/index.md # parent topics for sections must be an index.md file
-    - SI model with no demographics: notebooks/01_SI_nobirths_logistic_growth.ipynb # parent topics for sections must be an index.md file
-    - SI model with constant population demographics: notebooks/02_SI_wbirths_logistic_growth.ipynb
-    - SIS model with no demographics: notebooks/03_SIS_nobirths_logistic_growth.ipynb
-    - Outbreak size in the SIR model: notebooks/04_SIR_nobirths_outbreak_size.ipynb
-    - Average age at infection in the SIR model: notebooks/05_SIR_wbirths_age_distribution.ipynb
-    - Intrinsic periodicity of the SIR system: notebooks/06_SIR_wbirths_natural_periodicity.ipynb
-    - Exploring the critical community size of an SIR model: notebooks/07_SIR_CCS.ipynb
-    - The relationship between spatial coupling and incidence correlation in a 2-patch model: notebooks/08_2patch_SIR_wbirths_correlation.ipynb
-  - API reference: autoapi/*.md # include all the autogenerated markdown files
-  - Glossary: glossary.md
+  - Software overview:
+    - software-overview/index.md
+    - Demographics:
+      - software-overview/demographics/index.md
+      - software-overview/demographics/age_pyramid.ipynb
+      - software-overview/demographics/kmestimator.ipynb
+    - software-overview/migration.md
+    - software-overview/database.md
+    - software-overview/transmission.md
```

Comment:
It seems like transmission shouldn't live under the Demographics section--it looks like you have migration, DB, and transmission living outside that folder but still showing up in the TOC under that section.